packages/core/.storybook/preview.js

```diff
@@ -105,12 +105,56 @@ export const parameters = {
   },
 };
 
-export const decorators = [withCssResources];
+const defaultTheme = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : '';
+export const globalTypes = {
+  theme: {
+    name: 'Themes',
+    description: 'Available built in Clarity themes',
+    defaultValue: defaultTheme,
+    toolbar: {
+      items: [
+        { value: '', title: 'Light Theme' },
+        { value: 'dark', title: 'Dark Theme' },
+      ],
+    },
+  },
+  baseFont: {
+    name: 'Base Font',
+    description: 'Base Font for Document',
+    defaultValue: '125%',
+    toolbar: {
+      items: [
+        { value: '125%', title: 'Base 20px (Default)' },
+        { value: '100%', title: 'Base 16px' },
+      ],
+    },
+  },
+  // motion: {
+  //   name: 'Motion',
+  //   description: 'Clarity Animations',
+  //   defaultValue: '',
+  //   toolbar: {
+  //     items: [
+  //       { value: '', title: 'Enable Animations' },
+  //       { value: 'low-motion', title: 'Disable Animations' },
+  //     ],
+  //   },
+  // },
+};
+
+const themeDecorator = (story, { globals }) => {
+  const themes = `${globals.theme ? globals.theme : ''} ${globals.motion ? globals.motion : ''}`;
+  document.body.setAttribute('cds-theme', `${themes}`);
+  window.parent.document.body.setAttribute('cds-theme', `${themes}`);
+  document.documentElement.setAttribute('cds-base-font', `${globals.baseFont === '100%' ? '16' : ''}`);
+  return story();
+};
+
+export const decorators = [themeDecorator];
```

Comment:
Storybook decorator allows the theme to be toggled on the storybook manager UI as well as the demo iframe.